client/utils/formUtils.js

```diff
@@ -308,6 +311,18 @@ export const getValidation = (field, defaultValue) => {
 					},
 				},
 			}
+		case 'program':
```

Comment:
เราเอา required ออกไป เพราะ ตอน register student อยู่ในหน้า additional information for student ที่เป็น optional อะ

Reply:
อ่อ แต่ add scholarship มัน required

Reply:
งั้นต้องให้อันใดอันหนึ่งเขียน validation ทับ

Reply:
ต้องใส่ props register ทับ เช่น
register={{
						...register('cfpassword', {
							validate: {
								match: (value) => value === getValues('password') || getErrMsg('password', 'match'),
							},
						}),
					}}